Packs/Code42/Integrations/Code42/Code42.yml

```diff
@@ -284,7 +284,7 @@ script:
       description: The alert ID of the resolved alert.
       type: string
     description: Resolves a Code42 Security alert.
-  dockerimage: demisto/py42:1.0.0.6301
+  dockerimage: devdemisto/py42:1.0.0.9079
```

Comment:
TODO: Change this to released image when available

Reply:
devdemisto/py42:1.0.0.9079 is running locally on our server. We don't need to change it back for this PR, but before the release, it needs to be the public one.